frontend/app-development/features/administration/components/Administration.module.css

```diff
@@ -6,13 +6,14 @@
 
 .pageContainer {
   background-color: #e6eff8;
+  flex: 1;
   position: relative;
 }
 
 .pageContainer::before {
-  background-image: url('/designer/img/Altinn-studio.svg');
-  background-position: center 10px;
-  background-size: 100% auto;
+  background-image: url('/designer/img/overview-background.svg');
+  background-position: left 0;
```

Comment:
I wonder if we need to align the picture to the left and top? becuase i removed it i get the sane result.

Reply:
Removed! 👍